### ISSUE TITLE
No warning given when a folder cannot be moved to a new location (because it is open in the file system in some other program)

### ISSUE THREAD

**Issue by daniel347x** [Microsoft/vscode#71021]
Issue Type: <b>Bug</b>

Please see followup comment for the real issue.

--- ORIGINAL COMMENTS FOLLOW ---

Very odd.  In the VS Code explorer, I just moved 3 folders into another folder (prompted with the "Move Folder" dialog) just fine - the folders moved into their new location just fine.  These folders have some files in them (and some even had subfolders).  In fact, looking now, some of the files in some of these folders are currently open (as a tab) in the editor pane.

But trying repeatedly to drag & drop a folder named "Full", which contains both files and subfolders itself, into a new location - failed completely.  The folder just stays in its original location.  I am prompted with the "Move Folder" dialog and click OK, but the folder just stays put - it doesn't move.

Not sure what's up, but the issue sticks out like a sore thumb, so I'm sending this bug report!

VS Code version: Code 1.32.3 (a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4, 2019-03-14T23:43:35.476Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) Gold 6154 CPU @ 3.00GHz (36 x 3000)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|255.67GB (105.68GB free)|
|Process Argv||
|Screen Reader|no|
|VM|22%|

</details><details><summary>Extensions (40)</summary>

Extension|Author (truncated)|Version
---|---|---
markdown-toc|Ala|1.5.6
Bookmarks|ale|10.3.0
code-gnu-global|aus|0.2.2
markdown-extension-pack|bat|0.3.0
vscode-django|bat|0.19.0
markdown-emoji|bie|0.0.7
highlight-line-vscode|cli|0.0.1
c-cpp-compile-run|dan|0.5.0
markdown-table-prettify|dar|2.3.0
vscode-markdownlint|Dav|0.25.1
algorithm-mnemonics-vscode|dav|1.0.3
vscode-eslint|dba|1.8.2
make-hidden|dev|2.3.5
docs-markdown|doc|0.2.2
jupyter|don|1.1.5
python-extension-pack|don|1.4.0
docker-explorer|for|0.1.6
mdmath|goe|2.3.9
favorites|kdc|2.4.5
kite|kit|0.77.0
MagicPython|mag|1.1.0
markdown-shortcuts|mdi|0.11.0
vscode-todo-parser|min|1.9.1
vscode-kubernetes-tools|ms-|0.1.18
python|ms-|2019.2.5558
azure-account|ms-|0.8.0
cpptools|ms-|0.22.1
vsliveshare|ms-|0.3.1326
team|ms-|1.149.2
vetur|oct|0.17.0
vscode-docker|Pet|0.6.1
vscode-yaml|red|0.3.0
code-settings-sync|Sha|3.2.7
markdown-preview-enhanced|shd|0.3.13
vscodeintellicode|Vis|1.1.4
vscode-todo-highlight|way|1.0.4
debug|web|0.23.0
jinja|who|0.0.8
markdown-pdf|yza|1.2.0
markdown-all-in-one|yzh|2.1.1

(1 theme extensions excluded)

</details>

**Comment by daniel347x** [comment]
Follow-up: I have determined the cause of the issue, and RENAMED the issue appropriately.

I notice that the folder in question - "Full" - was open in a different application (XYPlorer, a Windows File Explorer replacement).

When I closed the folder in that application, and returned to VS Code to attempt the drag & drop (move) operation again in VS Code's Explorer pane, the folder moved just fine.

Therefore, the issue is one of confusion.  When a folder cannot be moved (and, presumably, a file as well), VS Code should give a meaningful warning message, such as "The folder is open in another application and can't be moved."

Thanks!
